css.js

```diff
@@ -1,36 +1,116 @@
 var loader = require("@loader");
 
+var isNode = typeof process === "object" && {}.toString.call(process) ===
+	"[object process]";
+var importRegEx = /@import [^uU]['"]?([^'"\)]*)['"]?/g;
+var resourceRegEx =  /url\(['"]?([^'"\)]*)['"]?\)/g;
+	
+var waitSeconds = (loader.cssOptions && loader.cssOptions.timeout)
+	? parseInt(loader.cssOptions.timeout, 10) : 60;
+var noop = function () {};
+var onloadCss = function(link, cb){
+	var styleSheets = document.styleSheets;
+	var i = styleSheets.length;
+	while( i-- ){
+		if( styleSheets[ i ].href === link ){
+			return cb();
```

Comment:
Call `detachEvents` here as well, that way if onloadCss finishes first we still remove the event listeners.